src/components/poursLists/KegSectionHeader.js

```diff
@@ -0,0 +1,78 @@
+// @flow
+
+import type { KegSection } from '../../stores/SectionPoursListStore';
+
+import * as React from 'react';
+import moment from 'moment';
+import { StyleSheet, Text, View } from 'react-native';
+import BeverageAvatar from '../../common/avatars/BeverageAvatar';
+import { COLORS, TYPOGRAPHY } from '../../theme';
+import { NULL_STRING_PLACEHOLDER } from '../../constants';
+
+const styles = StyleSheet.create({
+  beverageNameText: {
+    ...TYPOGRAPHY.secondary,
+  },
+  container: {
+    alignItems: 'center',
+    backgroundColor: COLORS.secondary2,
+    flexDirection: 'row',
+    paddingHorizontal: 10,
+    paddingVertical: 8,
+  },
+  dateContainer: {
+    paddingRight: 32,
+  },
+  dateLabelText: {
+    ...TYPOGRAPHY.small,
+    color: COLORS.textFaded,
+  },
+  datesContainer: {
+    flexDirection: 'row',
+  },
+  textContainer: {
+    flex: 1,
+    paddingLeft: 10,
+  },
+});
+
+type Props = {
+  section: KegSection,
+};
+
+class KegSectionHeader extends React.Component<Props> {
+  shouldComponentUpdate() {
+    return false;
+  }
+
+  render() {
+    const {
+      section: { beverage, floatedDate, tapDate },
+    } = this.props;
+
+    return (
+      <View style={styles.container}>
```

Comment:
This is Ok for now but I will want to minimize the amount of text. I'll play around with this a bit and talk with the beta testers.